api/candlepin-api-spec.yaml

```diff
@@ -4767,7 +4767,8 @@ paths:
           schema:
             type: array
             items:
-              $ref: '#/components/schemas/KeyValueParamDTO'
```

Comment:
We need to change this because, it wasn't working. Our client parse that object into string and send it in request in wrong way. Will already face this issue before me and came with this solution. I just "stole" his solution in his PR.